File: Python/Multiples of 3 or 5.py
def solution(number):
    sum = 0
    for x in range(number):
        # Multiplos de 3 y 5
        if x % 3 == 0 and x % 5 == 0:
            sum += x
        elif x % 3 == 0:
            sum += x
        elif x % 5 == 0:
            sum += x
        x -= 1
    return(sum)

File: Java/Multiples_of_3_or_5.java
package Java;

public class Multiples_of_3_or_5 {

    public int solution(int number) {
        var sum = 0;

        for (int x = number - 1; x >= 0; x--) {
            if (x % 3 == 0 && x % 5 == 0) {
                sum += x;
            }

            else if (x % 3 == 0) {
                sum += x;
            }

            else if (x % 5 == 0) {
                sum += x;
            }

        }

        return sum;
    }
}